Crash when auto complete via tab response from requests.get

When trying to auto complete via tab response from requests.get; crashes.

Reproduce steps + stacktrace below:
```
@ val r = requests.get("https://google.com.sg")
r: Response = Response(
  url = "https://www.google.com.sg/",
  statusCode = 200,
  statusMessage = "OK",
  data = <!doctype html><html itemscope="" itemtype="http://schema.org/WebPage" lang="en-MY"><head><meta content="text/html; charset=UTF-8" http-equiv="Content-Type"><meta content="/images/branding/googleg/1x/googleg_standard_color_128dp.png" itemprop="image"><title>Google</title><script nonce="amNMFn4qPMJYsuM10ThUGA">(function(){var _g={kEI:'pDTLZuXaD82k2roP4dfDmA8',kEXPI:'0,3700295,654,432,7,90,538567,2872,2891,561,7787,34680,30022,34266,145698,2,16737,23024,6699,41946,54825,2912,2,2,1,24626,2006,8155,23350,22436,48456,23980,76209,15816,1804,7734,27259,275,11814,1633,29278,14187,12896,5212675,999,124,21,5991287,2841236,97,6,32...

@ r.<tab>

@ r.Exception in thread "main" java.lang.AssertionError: assertion failed: missing implicit parameter of type requests.Response => geny.Readable after typer at phase parser
	at scala.runtime.Scala3RunTime$.assertFailed(Scala3RunTime.scala:8)
	at dotty.tools.dotc.typer.Implicits.inferImplicit(Implicits.scala:1043)
	at dotty.tools.dotc.typer.Implicits.inferImplicit$(Implicits.scala:819)
...
	at ammonite.repl.Repl.action$$anonfun$2(Repl.scala:211)
	at ammonite.util.Res$Success.flatMap(Res.scala:58)
	at ammonite.repl.Repl.action(Repl.scala:211)
	at ammonite.repl.Repl.loop$1(Repl.scala:218)
	at ammonite.repl.Repl.run(Repl.scala:233)
	at ammonite.Main.run(Main.scala:249)
	at ammonite.MainRunner.runRepl$$anonfun$1(MainRunner.scala:76)
	at ammonite.MainRunner.watchLoop(MainRunner.scala:57)
	at ammonite.MainRunner.runRepl(MainRunner.scala:76)
	at ammonite.AmmoniteMain$.main0(AmmoniteMain.scala:98)
	at ammonite.AmmoniteMain$.main(AmmoniteMain.scala:31)
	at ammonite.AmmoniteMain.main(AmmoniteMain.scala)
```

Details:
```
$ amm --version
Ammonite REPL & Script-Runner, 3.0.0-M2
```

System:
Mac M2 - Version 13.5 
```
$ uname -a
Darwin xxx 22.6.0 Darwin Kernel Version 22.6.0: Wed Jul  5 22:21:53 PDT 2023; root:xnu-8796.141.3~6/RELEASE_ARM64_T6020 arm64
```

Let me know if any other info needed.

--- a/amm/compiler/src/main/scala-3.0.0-3.3.1/dotty/ammonite/compiler/AmmCompletion.scala
+++ b/amm/compiler/src/main/scala-3.0.0-3.3.1/dotty/ammonite/compiler/AmmCompletion.scala
@@ -6,6 +6,7 @@ import dotty.tools.dotc.core.Contexts._
 import dotty.tools.dotc.core.Denotations.SingleDenotation
 import dotty.tools.dotc.core.Flags._
 import dotty.tools.dotc.core.Names.Name
+import dotty.tools.dotc.core.Phases
 import dotty.tools.dotc.core.Symbols.{Symbol, defn}
 import dotty.tools.dotc.interactive.{Completion, Interactive}
 import dotty.tools.dotc.util.SourcePosition
@@ -23,7 +24,7 @@ object AmmCompletion extends AmmCompletionExtras {
       path,
       dependencyCompleteOpt,
       enableDeep
-    )(using Interactive.contextOfPath(path))
+    )(using Interactive.contextOfPath(path).withPhase(Phases.typerPhase))
   }
 
   def computeCompletions(

--- a/amm/compiler/src/main/scala-3.3.2+/dotty/ammonite/compiler/AmmCompletion.scala
+++ b/amm/compiler/src/main/scala-3.3.2+/dotty/ammonite/compiler/AmmCompletion.scala
@@ -6,6 +6,7 @@ import dotty.tools.dotc.core.Contexts._
 import dotty.tools.dotc.core.Denotations.SingleDenotation
 import dotty.tools.dotc.core.Flags._
 import dotty.tools.dotc.core.Names.Name
+import dotty.tools.dotc.core.Phases
 import dotty.tools.dotc.core.Symbols.{Symbol, defn}
 import dotty.tools.dotc.interactive.{Completion, Interactive}
 import dotty.tools.dotc.util.SourcePosition
@@ -23,7 +24,7 @@ object AmmCompletion extends AmmCompletionExtras {
       path,
       dependencyCompleteOpt,
       enableDeep
-    )(using Interactive.contextOfPath(path))
+    )(using Interactive.contextOfPath(path).withPhase(Phases.typerPhase))
   }
 
   def computeCompletions(

--- a/amm/compiler/src/main/scala-3.4.2+/dotty/ammonite/compiler/AmmCompletion.scala
+++ b/amm/compiler/src/main/scala-3.4.2+/dotty/ammonite/compiler/AmmCompletion.scala
@@ -6,6 +6,7 @@ import dotty.tools.dotc.core.Contexts._
 import dotty.tools.dotc.core.Denotations.SingleDenotation
 import dotty.tools.dotc.core.Flags._
 import dotty.tools.dotc.core.Names.Name
+import dotty.tools.dotc.core.Phases
 import dotty.tools.dotc.core.Symbols.{Symbol, defn}
 import dotty.tools.dotc.interactive.{Completion, Interactive}
 import dotty.tools.dotc.util.SourcePosition
@@ -23,7 +24,7 @@ object AmmCompletion extends AmmCompletionExtras {
       path,
       dependencyCompleteOpt,
       enableDeep
-    )(using Interactive.contextOfPath(path))
+    )(using Interactive.contextOfPath(path).withPhase(Phases.typerPhase))
   }
 
   def computeCompletions(

--- a/amm/compiler/src/main/scala-3.5.0+/dotty/ammonite/compiler/AmmCompletion.scala
+++ b/amm/compiler/src/main/scala-3.5.0+/dotty/ammonite/compiler/AmmCompletion.scala
@@ -6,6 +6,7 @@ import dotty.tools.dotc.core.Contexts._
 import dotty.tools.dotc.core.Denotations.SingleDenotation
 import dotty.tools.dotc.core.Flags._
 import dotty.tools.dotc.core.Names.Name
+import dotty.tools.dotc.core.Phases
 import dotty.tools.dotc.core.Symbols.{Symbol, defn}
 import dotty.tools.dotc.interactive.{Completion, Interactive}
 import dotty.tools.dotc.util.SourcePosition
@@ -23,7 +24,7 @@ object AmmCompletion extends AmmCompletionExtras {
       path,
       dependencyCompleteOpt,
       enableDeep
-    )(using Interactive.contextOfPath(path))
+    )(using Interactive.contextOfPath(path).withPhase(Phases.typerPhase))
   }
 
   def computeCompletions(

